app/helpers/sheet/memberships/join_zusatzsektion.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+module Sheet::Memberships
+  class JoinZusatzsektion < Sheet::Base
+    self.parent_sheet = Sheet::Group
+
+    def initialize(*args)
+      super
+      @title = I18n.t('.title', scope: self.class.to_s.underscore)
```

Comment:
Rubocop complains about `Do not use instance variables in helpers.` Can't we just use `t('.title')` directly?

Reply:
Ich glaub wir machen das in den anderen sheets auch so, ich finde wir sollten die rubocop rule adaptieren, sodass sie nur für files mit suffix `_helper.rb` greift, habs bei https://github.com/hitobito/hitobito/issues/2715 ergänzt